src/server/game/Entities/Totem/Totem.cpp

```diff
@@ -66,6 +66,11 @@ void Totem::InitStats(uint32 duration)
 
         // set display id depending on caster's race
         SetDisplayId(GetOwner()->GetModelForTotem(PlayerTotemType(m_Properties->ID)));
+
+        SetOwnerGUID(GetOwner()->GetGUID());
```

Comment:
hack. Totems are not 'owned'. this applies only for charmed units

Reply:
Ye, I kind of figured that this wouldnt be the right way considering that other places in the code it is said not to call SetMinion method on totems. 

Nonetheless, I can say that right now, without setting owner guid, totems wont attack same faction in duels.

Reply:
yes, which is why I keep it unfixed. I wont tuctape stuff for the sake of ductaping it.

Reply:
Right, thats understandable. I have no idea, so I just try to fix stuff, make a PR and see what happens 😝